benchmark/scenarios/__init__.py

```diff
@@ -3,3 +3,8 @@
     ReadRequestLoadScenario, RateMeasurer, RequestRateTooLow,
     RequestRateNotReached, RequestOverload
 )
+from .write_request_load import (
+    WriteRequestLoadScenario, WRateMeasurer, WRequestRateTooLow,
+    WRequestRateNotReached, WRequestOverload, WDataseCreationTimeout
+)
```

Comment:
As for read request, the rate measurer is not a public object. Import it into the test module directly from `write_request_load`.